packages/generator-open-wc/README.md

```diff
@@ -24,24 +24,88 @@ or
 npx -p yo -p generator-open-wc -c 'yo open-wc:{generator-name}'
 ```
 
-## Available generators
 
-### Generators
-- open-wc:vanilla
-- open-wc:lint
-- open-wc:testing
-- open-wc:testing-upgrade
-- open-wc:publish-storybook
+## Scaffold generators
 
-### Optional generators
-- open-wc:testing-wallaby
+These generators help you kickstart a web component.
+You can use these in an empty folder to set up everything you need to get started immediately.
+
+Example usage:
+```bash
+mkdir foo-bar
+cd foo-bar
+yo open-wc:scaffold-vanilla
+```
+
+### Available scaffold generators:
+- `yo open-wc:scaffold-demoing`<br> 
+  If you already have a web component but no `/stories/` folder
```

Comment:
I thought the scaffold generators would bootstrap a new web component project for you? Also can we be more precise on what this gives you? Storybook, ESLint? Testing with Karma/Karma Browserstack? etc. (see upgrade generators)